validate-samples.sh

```diff
@@ -46,7 +46,7 @@ rm -f validate-output.txt &> /dev/null
 
 for schema in "devfile" "dev-workspace" "dev-workspace-template"
 do
-  schemaPath="./schemas/latest/with-markdown-descriptions/${schema}.json"
+  schemaPath="./schemas/latest/ide-targeted/${schema}.json"
```

Comment:
Can we do anything to prevent the inconsistency we have currently in master? Seems validate samples github action does not fail when it's supposed to 
![Screenshot_20201013_124237](https://user-images.githubusercontent.com/5887312/95844604-c2676480-0d51-11eb-810b-72ed93548a89.png)

Reply:
Fixed in commit https://github.com/devfile/api/pull/166/commits/35a4a0177a8fd17b8c78851b73a113a2029cd369